README.md

```diff
@@ -0,0 +1,49 @@
+
+# Project Name
+
+>spacex project .
+
+
+## Built With
+
+-react
+-redux
+**To get this project locally.**
+## Run this command on your terminal
+
+
+```
+git clone https://github.com/businesskaniki/spacex.git
+
+```
```

Comment:
- [ ] Kindly add the installation command `npm install`, and how to start the project locally `npm start `step by step.

Reply:
This is well resolved! ✔️